File: app/src/main/java/in/woloo/www/period_tracker/model/DailyLogWithTitle.java
package in.woloo.www.period_tracker.model;

import androidx.annotation.StringRes;

import java.util.List;

import in.woloo.www.dailylogscreen.models.DailyLogSymptoms;

public class DailyLogWithTitle {
    private List<DailyLogSymptoms> dailyLogSymptoms;
    @StringRes
    private int titleName;

    public DailyLogWithTitle(@StringRes int titleName, List<DailyLogSymptoms> dailyLogSymptoms) {
        this.dailyLogSymptoms = dailyLogSymptoms;
        this.titleName = titleName;
    }

    public List<DailyLogSymptoms> getDailyLogSymptoms() {
        return dailyLogSymptoms;
    }

    public void setDailyLogSymptoms(List<DailyLogSymptoms> dailyLogSymptoms) {
        this.dailyLogSymptoms = dailyLogSymptoms;
    }

    public int getTitleName() {
        return titleName;
    }

    public void setTitleName(int titleName) {
        this.titleName = titleName;
    }
}

File: app/src/main/java/in/woloo/www/more/period_tracker/model/DailyLogWithTitle.kt
package `in`.woloo.www.more.period_tracker.model

import androidx.annotation.StringRes
import `in`.woloo.www.more.dailylogscreen.models.DailyLogSymptoms

class DailyLogWithTitle(@StringRes titleName: Int, dailyLogSymptoms: List<DailyLogSymptoms?>) {
    private var dailyLogSymptoms: List<DailyLogSymptoms?>

    @StringRes
    var titleName: Int

    init {
        this.dailyLogSymptoms = dailyLogSymptoms
        this.titleName = titleName
    }

    fun getDailyLogSymptoms(): List<DailyLogSymptoms?> {
        return dailyLogSymptoms
    }

    fun setDailyLogSymptoms(dailyLogSymptoms: List<DailyLogSymptoms>) {
        this.dailyLogSymptoms = dailyLogSymptoms
    }
}
